### ISSUE TITLE
Random isn't initialized properly

### ISSUE THREAD

**Issue by mikehofm** [Unity-Technologies/Unity.Mathematics#78]
Consider the following code:
```
for (uint i = 1; i < 10; i++)
	Debug.Log(new Unity.Mathematics.Random(i).NextFloat());
```
It outputs the following:
```
6.29425E-05
0.000125885
0.0001888275
0.00025177
0.0003147125
0.000377655
0.0004405975
0.00050354
0.0005664825
```
This output does not look random. This is because the Xorshift PRNG Random() uses is supposed to be initialized with different type of PRNG, such as SplitMix. Wikipedia has an example: https://en.wikipedia.org/wiki/Xorshift#Initialization.

**Comment by unpacklo** [comment]
This will be addressed with this PR, whenever it lands https://github.com/Unity-Technologies/Unity.Mathematics/pull/128

**Comment by fsilvestrim** [comment]
Quick question, was this really fixed? because I'm still having similar issues.

**Comment by unpacklo** [comment]
Yes, this was fixed with PR #128 but unless you're directly using the GitHub repo copy of Mathematics in your own project, you won't see the fix since we haven't done a release yet. If you really need the fix on your end, you could point your project to your GitHub copy.
